src/browser/components/SignpostImg.react.js

```diff
@@ -16,13 +16,6 @@ export default class SignpostImg extends Component {
     delayStart: PropTypes.number
   }
 
-  static defaultProps = {
```

Comment:
Sorry, i was thinking it wasn't used anywhere but now i checked and see that React class is using it without help ;)